release.md

```diff
@@ -0,0 +1,93 @@
+# Release procedure
+
+## Branching
+
+1. Checkout `master` branch.
+   
+    `git checkout master`
+   
+1. Make sure you have the latest upstream changes.
+
+    `git pull upstream master`
+
+1. Create a new release branch.
+
+    `git checkout -b v1.x`
+   
+1. Push branch to the upstream.
+
+    `git push upstream v1.x`
```

Comment:
Creating a local branch may be prone to accidentally including some changes. I think directly pushing the sha would be safer.
```
git fetch upstream master
git push upstream upstream/master:vX.X
```